spec/spec_helper.rb

```diff
@@ -10,7 +12,9 @@
 
 require 'rspec'
 require 'facterdb'
+# rubocop:disable Style/MixinUsage
 include FacterDB
+# rubocop:enable Style/MixinUsage
 
 RSpec::Matchers.define_negated_matcher :not_be_nil, :be_nil
 RSpec::Matchers.define_negated_matcher :not_be_empty, :be_empty
```

Comment:
A bit below is `project_dir` which can be safely simplified to `File.dirname(__dir__)`

Reply:
:+1: